### DISCUSSION TITLE
Create a discord

### DISCUSSION THREAD

**Discussion by OpenSauce** [FerretDB/FerretDB#25] [General]
Create a developmentt discord for the project

[a comment by an automated account is not shown]

**Comment by OpenSauce** [comment]
Any is fine :) Discord was just an example

**Comment by OpenSauce** [comment]
Any update on this? @mangodb-bot Saves me asking dumb questions on merge requests 😄

[a comment by an automated account is not shown]

**Comment by ptrfarkas** [comment]
Just to have it here as well, our community Slack is at [ferretdb.slack.com](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A). Feel free to join!
